.github/workflows/deploy.yml

```
@@ -0,0 +1,32 @@
+name: Deploy
+
+on:
+  push: {branches: [main]}
+  schedule: [{cron: "15 10 * * *"}]
+  workflow_dispatch: {}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pages: write
+      id-token: write
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 24
+          cache: npm
```

```suggestion
          cache: yarn
```

(I'm not sure whether that's correct?)